portalnet/src/overlay/protocol.rs

```diff
@@ -70,7 +71,7 @@ pub struct OverlayProtocol<TContentKey, TMetric, TValidator, TStore> {
     /// The overlay routing table of the local node.
     kbuckets: Arc<RwLock<KBucketsTable<NodeId, Node>>>,
     /// The subnetwork protocol of the overlay.
-    protocol: ProtocolId,
+    protocol: Subnetwork,
```

Comment:
I left the original `protocol` var alone here since it's used extensively in our logs & comments. At some point, these two phrases are interchangeable, so i'm not sure it's worth updating all the `protocol` usage in our comments to `subnetwork`